src/seth/libexec/seth/seth---to-int256

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+### seth---to-int256 -- convert an integer into int256 value
+### Usage: seth --to-int256 <value>
+set -e
+export BC_LINE_LENGTH=5000
+seth --to-uint256 $(bc <<< "(2^256 * ($1 < 0)) + $1")
```

Comment:
[`shellcheck`](http://shellcheck.net/) gives a warning about missing quotes around the process substitution - could you do that?

I can recommend installing a `shellcheck` plugin locally btw, and I plan to add it to the CI pipeline.

Reply:
2nded, @MrChico I recommend installing `shellcheck` and `(add-hook 'sh-mode-hook 'flycheck-mode)`.

Reply:
It's now run in Travis https://github.com/dapphub/dapptools/pull/175

Reply:
Thanks, should be fixed now